### PR TITLE
RunsOn entry

### LINKED ISSUE #1: Add an entry for https://runs-on.com
Requires some reading and investigation first.

### PR DESCRIPTION
**Pull request by neysofu**
Closes #1.

cc @crohr, feel free to provide feedback!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -138,6 +120,38 @@ Notable features:
...
 [^5c]: https://github.com/ubicloud/ubicloud/blob/main/routes/web/webhook/github.rb
 
+### RunsOn
+
+[RunsOn](https://runs-on.com/) is a self-hosted GitHub Actions runner tool that runs on your own AWS account. It's **free for non-commercial usage**, and comes at a €300 flat yearly fee for commercial projects.
+
+Pricing [here](https://runs-on.com/).
+
+Notable features:
+- **Easy install with a QuickFormation template** (video guide available).
```
Comment (crohr) [inline README.md:130]: CloudFormation

File: README.md [hunk 2]
```diff
@@ -138,6 +120,38 @@ Notable features:
...
+- Cheap, flat-rate pricing with an easy-to-use [pricing calculator](https://runs-on.com/calculator/) that you can use to calculate your spending.
+- ARM & GPU support.
```
Comment (crohr) [inline README.md:132]: X64, ARM & GPU support

File: README.md [hunk 3]
```diff
@@ -138,6 +120,38 @@ Notable features:
 [^5b]: https://www.ubicloud.com/blog/ubicloud-hosted-arm-runners-100x-better-price-performance
 [^5c]: https://github.com/ubicloud/ubicloud/blob/main/routes/web/webhook/github.rb
 
+### RunsOn
+
+[RunsOn](https://runs-on.com/) is a self-hosted GitHub Actions runner tool that runs on your own AWS account. It's **free for non-commercial usage**, and comes at a €300 flat yearly fee for commercial projects.
+
+Pricing [here](https://runs-on.com/).
+
+Notable features:
```
Comment (crohr) [inline README.md:129]: I would also add the "1-line change" sentence as for the others, and mention the runtime flexibility in the choice of instance types, disk size, ssh access, and most importantly custom base images

**Review by crohr — COMMENTED** [review]
I left a few comments for you to consider. Thanks!

File: README.md [hunk 4]
```diff
@@ -138,6 +120,38 @@ Notable features:
...
+- **Easy install with a QuickFormation template** (video guide available).
+- Cheap, flat-rate pricing with an easy-to-use [pricing calculator](https://runs-on.com/calculator/) that you can use to calculate your spending.
+- ARM & GPU support.
+- Up to 256 vCPUs per runner, by far the most of any provider.
+- Although actively maintained and developed as of Feb. '24, it's built by just one developer ([Cyril Rohr](https://runs-on.com/about/)), so long-term maintenance is a concern. ⚠️
```
Comment (crohr) [inline README.md:134]: (The code is public though). But yes I need to work a bit on the license terms to allow for modifications by the buyer when a license has been bought.

File: README.md [hunk 5]
```diff
@@ -8,15 +8,16 @@ GitHub Actions are pretty damn cool, but lord knows the official runners are slo
...
 	- [Table of contents](#table-of-contents)
 	- [List of providers](#list-of-providers)
 		- [Namespace ⭐](#namespace-)
 		- [BuildJet ⭐](#buildjet-)
 		- [Actuated ⭐](#actuated-)
 		- [WarpBuild](#warpbuild)
-		- [Cirun](#cirun)
 		- [Ubicloud](#ubicloud)
+		- [RunsOn](#runson)
+		- [Cirun](#cirun)
```
Comment (irogerthat) [inline README.md:20]: I am curious, why this ordering is changed? Also what's the purpose of ⭐ in front of the first 3?

**Comment by crohr** [comment]
@neysofu anything I can do to help close this issue? Thanks!

File: README.md (hunk 2 above)
Comment (neysofu) [reply README.md:132]: I'm omitting X86-64 as that's something that all providers currently support, although this is good feedback and it arguably should be made more explicit in a future PR.

File: README.md (hunk 4 above)
Comment (neysofu) [reply README.md:134]: Nice. In the meantime I added a note to highlight the fact that it's open source. The bus factor is a factor that I'm taking into consideration for all providers though, so while last, I think it's reasonable to keep it in.

File: README.md (hunk 3 above)
Comment (neysofu) [reply README.md:129]: Good stuff, I'll make some modifications.

**Comment by neysofu** [comment]
I've pushed some changes, kindly let me know what you think when you have got the time. Thanks!

**Review by crohr — APPROVED** [review]
That works for me, thanks!

**Pull request merged**